data/sql/updates/pending_db_world/ethereum-prisoner-update.sql

```diff
@@ -0,0 +1 @@
+UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask`|1|2|8|16|32|64|256|512|1024|2048|4096|8192|16384|65536|131072|262144|524288|2097152|4194304|67108864|536870912|1073741824 WHERE `entry` IN (22827, 22826, 22825, 20888, 22828, 22920);
```

Comment:
```suggestion
UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask`|1|2|8|16|32|64|256|512|1024|2048|4096|8192|65536|131072|524288|4194304|8388608|67108864|536870912 WHERE `entry` IN (22827, 22826, 22825, 20888, 22828, 22920);
```

add horror
remove enrage, infected, shield, bleed